dev-images/docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM fedora:26
+
+ENV BUILD_SCRIPT        build_leapp_environment.sh
+ENV BUILD_SCRIPT_URL    http://leapp-to.github.io/files/$BUILD_SCRIPT
+ENV HOME                /root
+ENV LEAPP_HOME          $HOME/leapp
+
+
+RUN dnf update -y
+RUN dnf -y install 'dnf-command(copr)' sudo libvirt docker && \
```

Comment:
Join these two together so that the `clean` operation cleans up the `update` operation as well.